sdk/js/src/solana/wormhole/accounts/guardianSet.ts

```diff
@@ -0,0 +1,87 @@
+import {
+  Connection,
+  PublicKey,
+  Commitment,
+  PublicKeyInitData,
+} from "@solana/web3.js";
+import {
+  ETHEREUM_KEY_LENGTH,
+  deriveAddress,
+  getAccountData,
+} from "../../utils";
+
+export function deriveGuardianSetKey(
+  wormholeProgramId: PublicKeyInitData,
+  index: number
+): PublicKey {
+  return deriveAddress(
+    [
+      Buffer.from("GuardianSet"),
+      (() => {
+        const buf = Buffer.alloc(4);
+        buf.writeUInt32BE(index);
+        return buf;
+      })(),
+    ],
+    wormholeProgramId
+  );
+}
+
+export async function getGuardianSet(
+  connection: Connection,
+  wormholeProgramId: PublicKeyInitData,
+  index: number,
+  commitment?: Commitment
+): Promise<GuardianSetData> {
+  return connection
+    .getAccountInfo(deriveGuardianSetKey(wormholeProgramId, index), commitment)
+    .then((info) => GuardianSetData.deserialize(getAccountData(info)));
+}
+
+export class GuardianSetData {
+  // pub struct GuardianSetData {
```

Comment:
commented out code

Reply:
Removed this and other spots: https://github.com/wormhole-foundation/wormhole/pull/1375/commits/aaa480b0b721220f307a96467bafcea3e6b75172